index.js

```diff
@@ -106,22 +106,126 @@ function selectionChanger(selection,operator,endoperator){
 }
 
 editor.addKeyMap({
-    // bold
-    'Ctrl-B': function(cm) {
-        cm.replaceSelection(selectionChanger(cm.getSelection(),'**'));
-    },
-    // italic
-    'Ctrl-I': function(cm) {
-        cm.replaceSelection(selectionChanger(cm.getSelection(),'_'));
-    },
-    // code
-    'Ctrl-K': function(cm) {
-        cm.replaceSelection(selectionChanger(cm.getSelection(),'`'));
-    },
-    // keyboard shortcut
-    'Ctrl-L': function(cm) {
-        cm.replaceSelection(selectionChanger(cm.getSelection(),'<kbd>','</kbd>'));
+  // bold
+  'Ctrl-B': function(cm) {
+    var selection = cm.getSelection();
+    cm.replaceSelection('**' + selection + '**');
```

Comment:
This change takes away the existing functionality where you could use CTRL+B to toggle bold. This would always add more text. For example, pressing the hotkey twice on 'hello' would result in:

```
=> hello
=> **hello**
=> ****hello****
```

Arguably, the first approach wasn't foolproof either, but this implementation is immediately broken.